public/app/plugins/datasource/prometheus/legend.test.ts

```diff
@@ -18,13 +18,31 @@ describe('renderLegendFormat()', () => {
 
     // not sure if this is expected... but current behavior
     expect(renderLegendFormat('{{ a }}', labels)).toEqual('AAA');
+    expect(renderLegendFormat('{{             a                        }}', labels)).toEqual('AAA');
+  });
+
+  it('Missing label', () => {
+    expect(renderLegendFormat('value: {{c}}', labels)).toEqual('value: {{c}}');
+  });
+
+  it('Nested brackets', () => {
+    expect(renderLegendFormat('{{{a}}}', labels)).toEqual('{AAA}');
```

Comment:
this is one behavior that has changed in this PR. it's unclear if this is the expected behavior, but the regex now matches the nested `{{a}}`

```
input: {{{a}}}
output from main: {a}
output from this PR: {AAA}
```